resources/lang/en/items.php

```diff
@@ -26,10 +28,14 @@
         'price'     => 'Price of the item',
         'size'      => 'Size, Weight, Dimensions',
         'type'      => 'Weapon, Potion, Artefact',
+        'parent'    => 'ID of the parent item'
     ],
     'show'          => [
         'tabs'  => [
             'inventories'   => 'Inventories',
         ],
     ],
+    'hints'         => [
+        'items'    => 'A web of interlocking items can be built using the Parent Item field.',
```

Comment:
I can tell this was copied from quests, where it makes sense. However for items, we can simply use "Organise items by using the parent item field."